src/controllers/auth.js

```diff
@@ -227,7 +228,7 @@ const obtainAccessToken = async (req, res) => {
     const payload = {
       username: user.username,
       userId: user.id,
-      roleId: usersRole.role_id,
+      roleId: usersRole[0].role_id,
```

Comment:
Why are we getting the first role?

Reply:
We're (currently) following a model where a user only has one role.

I see our error.  If a user only has one role, we don't need `_user_roles` table - we could just make `role` an addribute of `_user`.

@thevahidal - do you believe we should allow a user to have multiple roles?